codebuild_specs/e2e_workflow_generated.yml

```diff
@@ -145,7 +136,7 @@ batch:
       env:
         variables:
           TEST_SUITE: src/__tests__/diagnose.test.ts|src/__tests__/hooks-a.test.ts|src/__tests__/mock-api.test.ts
-          CLI_REGION: us-west-2
+          CLI_REGION: us-east-2
```

Comment:
why changing region?

Reply:
That's default behavior of `yarn split-e2e-tests-codebuild` . (which we sometimes revert manually, but in this case it doesn't make much sense since tests got reshuffled a bit more than usually).

Reply:
I.e. this place https://github.com/aws-amplify/amplify-cli/blob/9034f865172fbc754a37c00783490ea972e02410/scripts/split-e2e-tests-codebuild.ts#L157